js/__internal/grids/grid_core/data_controller/module.ts

```diff
@@ -1256,7 +1258,7 @@ export class DataController extends ControllerWithDataMixin {
     return dataSourceAdapter;
   }
 
-  isLocalStore(store) {
+  isLocalStore(store?) {
     store = store || this.store();
```

Comment:
By our code convention let's change it:

```suggestion
    store ||= this.store();
```

Or can we use the default parameter here?
```typescript
isLocalStore (store = this.store()) {
 // other code.
}